grant-project/grant-monorepo
# Use UserAvatar on Review

### What this does
- use `UserAvatar` on `Review`
- add missing map keys

### Test
- create flow to review section, check that blockies render for users w/o image avatars
- make sure that there are no error warnings issued about missing keys
